inputFiles/pygeos/run_sedov_problem.py

```diff
@@ -0,0 +1,80 @@
+
+import numpy as np
+from mpi4py import MPI
+import pygeosx
+from pygeosx_tools import wrapper
+from geosx_xml_tools.main import preprocess_parallel
```

Comment:
do you need this? It does not seem to be used.

Reply:
No, I missed cleaning up the import list when I removed the pre-processor from this example